src/LogController.php

```diff
@@ -40,7 +40,7 @@ public function tail($file, Request $request)
     {
         $offset = $request->get('offset');
 
-        $viewer = new LogViewer($file);
+        $viewer = new LogViewer($file.".log");
```

Comment:
since you already add `.log` in `LogViewer::getFilePath()`, this line may cause `laravel.log.log` issue